mat64/triangular.go

```diff
@@ -115,7 +115,18 @@ func (t *TriDense) Dims() (r, c int) {
 }
 
 func (t *TriDense) Triangle() (n int, upper bool) {
-	return t.mat.N, t.mat.Uplo == blas.Upper
+	return t.mat.N, t.isUpper()
```

Comment:
Still thinking, but I could add ��!t.isZero() &&��. It feels unsatisfying, and I'd prefer to just document that Triangle may not be called on a zero value, but I'll bend.